docs/DeveloperGuide.md

```diff
@@ -198,6 +198,33 @@ The following sequence diagram below shows how `Model` and `LogicManger` compone
 #### Proposed Changes
 Include `Person` meetings as a search field. A user can supply a given date and will return all `Person` that have a meeting starting or ending on the specified date.
 
+### AddTag Feature
+This feature allows users to add `tags` to a `person` within the contact list, without the need to use the `edit` command.
+
+This feature is implemented through the `AddTagCommand` and the `AddTagCommandParser` which extends `Command` and `Parser` respectively. The `AddTagCommandParser` takes in an `index` and the `tags` to add to a person. If both are supplied and valid, they are passed into the `AddTagCommand`.
+
+<div markdown="span" class="alert alert-info">:information_source: **Note:** This feature can be used in conjuncture with the find feature, where it can be called on a list filtered by find.
+</div>
+
+The current allowed formats for required fields are as follows:
+- `Index` : an `Integer` within the range greater than 0, smaller than the size of the currently displayed contact list.
+- `Tags` : one or more alphanumeric characters with no spaces between them.
+
+The following sequence diagram shows how the `AddTag` interacts with `Logic` and `Model`.
+
+![AddTagSequenceDiagram](images/AddTagSequenceDiagram.svg)
+
+1. The user keys in `addtags 1 t/validTag1 t/validTag2` to add 2 valid tags to the `person` at the first `index`.
+2. The `AddTagCommandParser` checks that the `index` and `tags` are valid, then returns a new `AddTagCommand` with the corresponding index and set of tags.
+3. The `LogicManager` then executes the `AddTagCommand`.
+4. The `AddTagCommand` finds the `Person` to add tags to using `Index` and creates a new `Person` with the added tags.
+5. `AddTagCommand` then calls the `setPerson(person, personWithAddedTags)` method to set the old `Person` to the newly created `Person`.
+6. `AddTagCommand` then calls `updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS)` to update `UI` to display the person with the newly added `Tags`.
+7. `CommandResult` is then returned to `LogicManager`.
+
+#### Proposed Changes
+Allow users to add tags to multiple people at once.
```

Comment:
Is there a better phrasing for this? I think it should not be proposed changes but proposed future implementations? Given that our command allows our command to add multiple tags to a single person, where this command implies adding a single (or multiple tag) to multiple people. It could possibly be considered a proposed change if the intention is to allow adding multiple tags to multiple people but I think this should be stated more clearly so that it is more clear that it is a change to this feature?